grpc-locations/src/main/kotlin/io/quarkus/sample/superheroes/location/grpc/LocationGrpcService.kt

```diff
@@ -0,0 +1,73 @@
+package io.quarkus.sample.superheroes.location.grpc
+
+import io.grpc.Status
+import io.grpc.stub.StreamObserver
+import io.smallrye.common.annotation.Blocking
+import io.quarkus.grpc.GrpcService
+import io.quarkus.logging.Log
+import io.quarkus.sample.superheroes.location.grpc.LocationsGrpc.LocationsImplBase
+import io.quarkus.sample.superheroes.location.mapping.LocationMapper
+import io.quarkus.sample.superheroes.location.service.LocationService
+
+@GrpcService
+class LocationGrpcService(private val locationService: LocationService) : LocationsImplBase() {
```

Comment:
The indentation looks odd (might be a github UI issue)

Reply:
What about the indentation looks odd? Even in the GitHub UI it looks ok to me...